builder/Dockerfile.centos-7

```diff
@@ -73,4 +73,5 @@ ENV R_RD4PDF="times,hyper"
 
 COPY package.centos-7 /package.sh
 COPY build.sh .
+COPY patches /patches
```

Comment:
Based on previous notes, CentOS 7 was supposedly unaffected, and I do see an installer up at https://cdn.rstudio.com/r/centos-7/pkgs/R-3.3.0-1-1.x86_64.rpm

Reply:
Oh, OK. OTOH, it also does not hurt to do it for CentOS 7 as well, and then it is simpler if we do it everywhere.

Reply:
Sure, that's a fine reason to keep it in then.